src/base/system_parameter.c

```diff
@@ -633,6 +633,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_CTE_MAX_RECURSIONS "cte_max_recursions"
 
+#define PRM_NAME_JSON_SHOW_CALLSTACK "log_json_allocations"
```

Comment:
What about renaming it as "json_log_allocations"? 
We may add others with `json_` prefix.

Please also rename JSON_SHOW_CALLSTACKs.